cores/rp2040/SerialPIO.h

```diff
@@ -41,6 +41,9 @@ class SerialPIO : public HardwareSerial {
     void begin(unsigned long baud, uint16_t config) override;
     void end() override;
 
+		void setInverted(bool i);
```

Comment:
Can we make this `setInverted(bool invTx = true, invRx = true)`?  Even if it's not implemented I can throw RX together by inspection quickly and have a good shot at it working.

Reply:
good idea. I'll have it ignore rx for now.